src/ImageHelper.php

```diff
@@ -393,6 +393,23 @@ public static function get_sideloaded_file_loc($file)
         $file = \parse_url($file);
         $path_parts = PathHelper::pathinfo($file['path']);
         $basename = \md5($filename);
+
+        /**
+         * Filters basename for sideloaded files.
+         * @since 2.0.1
+         * @example
+         * ```php
+         * // Change the basename used for sideloaded images.
+         * add_filter( 'timber/image_helper/sideload_image/basename', function ($basename, $path_parts) {
```

Comment:
Damn, forgot this one :)

```suggestion
         * add_filter( 'timber/image_helper/sideload_image/basename', function ($basename, $path_parts) {
```

Reply:
Fixed in https://github.com/timber/timber/pull/2897/commits/141eeb10495881cfe56e8140886ae56f308e73d2